src/common/pagination.py

```diff
@@ -148,13 +210,21 @@ def decode_pagination_token(
 
 
 def encode_pagination_token(
-    pagination_token: str, encrypted: bool, encryption_client: EncryptionClient = None
+    pagination_token: str,
```

Comment:
I think you're passing in `last_evaluated_key` as `pagination_token`, right? 

```
        response["NextToken"] = encode_pagination_token(
             last_evaluated_key,
```

Which is taken directly from the response:

```
last_evaluated_key = response.get("LastEvaluatedKey")
```

Which (I think) [is defined as a dict](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb.html#DynamoDB.Table.scan).

Following through the code, I don't think that actually breaks, because the `context.package` takes `Any` as the token type, but seems like a broken link typing-wise here.

```suggestion
    pagination_token_data: Any,
```

Reply:
yes, and it looks like the call to `_encode...` similarly has the parameter misnamed